commit 31970608a6d3796c3adbfbfd379fa3092de65c5d
Author: Kees Cook <kees@kernel.org>
Date:   Mon Sep 26 12:45:32 2022 -0700

    overflow: Fix kern-doc markup for functions
    
    Fix the kern-doc markings for several of the overflow helpers and move
    their location into the core kernel API documentation, where it belongs
    (it's not driver-specific).
    
    Cc: Jonathan Corbet <corbet@lwn.net>
    Cc: linux-doc@vger.kernel.org
    Cc: linux-hardening@vger.kernel.org
    Reviewed-by: Akira Yokosawa <akiyks@gmail.com>
    Signed-off-by: Kees Cook <keescook@chromium.org>

diff --git a/Documentation/driver-api/basics.rst b/Documentation/driver-api/basics.rst
index 3e2dae954898..4b4d8e28d3be 100644
--- a/Documentation/driver-api/basics.rst
+++ b/Documentation/driver-api/basics.rst
@@ -107,9 +107,6 @@ Kernel utility functions
 .. kernel-doc:: kernel/panic.c
    :export:
 
-.. kernel-doc:: include/linux/overflow.h
-   :internal:
-
 Device Resource Management
 --------------------------